pkg/controller/job/job_controller_test.go

```diff
@@ -5305,7 +5321,6 @@ func TestFinalizerCleanup(t *testing.T) {
 	}); err != nil {
 		t.Errorf("Waiting for Pod to get the finalizer removed: %v", err)
 	}
-
```

Comment:
revert, just to keep the diff minimal

Reply:
Can you also please handle this nit?

Reply:
oh sorry @mimowo, this one slipped, should be fixed now